README.md

```diff
@@ -36,6 +36,8 @@ Categories:
 
 Note that the `Prerelease` flag can be omitted for non-prerelease packages and controls whether NuGet will attempt to download a prerelease version of the package when generating the site.
 
+Please note that addin authors do not have to manually create this YAML as there is an automated process that scans Nuget.org twice per day to find all addins that follow the recommended naming convention (which is `Cake.xxx` where xxx describes the functionality provided by the addin) and generate the appropriate YAML content based on the metadata for the nuget package.
```

Comment:
I would put this at the beginning of the section and maybe format as blockquote to make it better visible